spec/reporter-spec.js

```diff
@@ -169,6 +169,16 @@ describe("Reporter", () => {
         {name: 'bundled-2', path: '/Applications/Atom.app/Contents/Resources/app.asar/node_modules/bundled-2', metadata: {version: '1.2.0'}},
       ]
 
+      const bundledPackages = [
+        'bundled-1',
+        'bundled-2'
+      ]
+
+      const packageDirPaths = ['/Users/user/.atom/packages']
```

Comment:
It may be worth it in a follow-up PR to convert all of these hardcoded paths to use `path.join` to ensure that everything works correctly on Windows as well.

Reply:
:+1: I can do that